src/coreclr/jit/compiler.cpp

```diff
@@ -2284,6 +2284,11 @@ void Compiler::compSetProcessor()
 #ifdef TARGET_XARCH
     if (!compIsForInlining())
     {
+        if (canUseEvexEncoding())
+        {
+            codeGen->GetEmitter()->SetUseEVEXEncoding(true);
```

Comment:
Would be good to assert `canUseVexEncoding()` is true here.

Reply:
Assumption being that, if we can useEvexEncoding, then  implicitly useVexEncoding is always true?

Reply:
I think this comment got placed in the wrong space by GitHub.

There'd be no need to assert `canUseEvexEncoding()` it given its in an `if (canUseEvexEncoding())` block.